src/edge/views.py

```diff
@@ -1,29 +1,51 @@
 import json
+import os
 import random
+import tempfile
 
 from django.core.urlresolvers import reverse
-from django.http import HttpResponse
+from django.http import HttpResponse, JsonResponse
 from django.views.generic.base import View
+from django.views.decorators.http import require_http_methods
 from django.shortcuts import get_object_or_404
 from django.db.models import Q
 from django.db import transaction
 
 from edge.models import Fragment, Genome, Operation
 from edge.io import IO
+from edge import import_gff
+from edge.tasks import build_genome_blastdb
 
 
 def genome_export(request, genome_id):
     get_genome_or_404(genome_id)
     io = IO(Genome.objects.get(pk=genome_id))
-
     response = HttpResponse(content_type='text/plain')
     response['Content-Disposition'] = "attachment; filename=\"g%s.gff\"" % genome_id
     io.to_gff_file(response)
     return response
 
 
+@require_http_methods(["POST"])
+def genome_import(request):
+    res = {
+      'imported_genomes': [],
+    }
+    for name in request.FILES:
```

Comment:
this is more a matter of taste, but I find it much easier to read code with period line breaks in it to separate "paragraphs" or "thoughts". In here, to increase legibility, I'd stick extra lines after the dict literal, and before the `return`,  and around each individual "component" of the for loop:  the gff writing/deletion, the BLASTdb task creation, and the call to `JsonResponse`.